src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -10,13 +10,6 @@
 import java.util.HashSet;
 
 public class ProjectData {
-  private final String PROPERTY_NAME = "name";
-  private final String PROPERTY_CREATOR = "creator";
-  private final String PROPERTY_DESCRIPTION = "description";
-  private final String PROPERTY_TASKS = "tasks";
-  private final String PROPERTY_ADMINS = "admins";
-  private final String PROPERTY_MEMBERS = "members";
```

Comment:
I thought the constants were a good idea.  Curious why you switched to hard coded property names

Reply:
The constants would appear in the json that was created in the HomePageServlet. It technically doesn't affect functionality but it looked pretty out of place

Reply:
I see.  I did a quick search, looks like tagging them with the "transient" keyword would prevent serialization into json.  I'm not sure if tagging them as "static" would as well